### PR TITLE
docs: logits -> probs in Accuracy metric documentation

### PR DESCRIPTION
**Pull request by Kulikovpavel**
## What does this PR do?

Documentation fix for Accuracy metric

There should be probs instead of logits as input.

## Before submitting
- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes [if needed]?
- [x] Did you write any new necessary tests [no need for typos, docs]? 
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:  

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [ ] **Check that target branch and milestone are aligned!**
 

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
@Kulikovpavel pls return the checklist to the PR description, it is there on purpose...

File: docs/source/metrics.rst [hunk 1]
```diff
@@ -33,10 +33,10 @@ The example below shows how to use a metric in your ``LightningModule``:
         self.accuracy = pl.metrics.Accuracy()
 
     def training_step(self, batch, batch_idx):
-        logits = self(x)
+        probs = self(x)
```
Comment (awaelchli) [inline docs/source/metrics.rst:36]: How did you derive that it has to be probabilities?

Comment (Kulikovpavel) [reply docs/source/metrics.rst:36]: There is a code like this in Accuracy class
```
    if preds.ndim == target.ndim and preds.is_floating_point():
        # binary or multilabel probablities
        preds = (preds >= threshold).long()
    return preds, target
```

Default threshold == 0.5, so it is probabilities, in case of logits it should be 0.

Comment (awaelchli) [reply docs/source/metrics.rst:36]: and it also depends on the shape of the tensor, and whether it is binary or multiclass right? these values are unknown in this particular example so it could really be anything :)

Comment (Kulikovpavel) [reply docs/source/metrics.rst:36]: with multiclass code:
https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/metrics/utils.py#L46
```
def _input_format_classification(
    preds: torch.Tensor, target: torch.Tensor, threshold: float = 0.5
) -> Tuple[torch.Tensor, torch.Tensor]:
    """Convert preds and target tensors into label tensors"""

    ...

    if preds.ndim == target.ndim + 1:
        # multi class probabilites
        preds = torch.argmax(preds, dim=1)

    if preds.ndim == target.ndim and preds.is_floating_point():
        # binary or multilabel probablities
        preds = (preds >= threshold).long()
    return preds, target
```

It can be integers (exact predictions), or probabilities [0,1] with default threshold == 0.5. Logits are [-inf, +inf]

File: docs/source/metrics.rst [hunk 2]
```diff
@@ -33,10 +33,10 @@ The example below shows how to use a metric in your ``LightningModule``:
         self.accuracy = pl.metrics.Accuracy()
 
     def training_step(self, batch, batch_idx):
```
Comment (rohitgr7) [inline docs/source/metrics.rst:35]: also missing `x, y = batch` I guess.

File: docs/source/metrics.rst (hunk 1 above)
Comment (rohitgr7) [reply docs/source/metrics.rst:36]: let's just call it `preds` since it can be anything??

Comment (Kulikovpavel) [reply docs/source/metrics.rst:36]: Sure, will do, just not logits

**Review by rohitgr7 — APPROVED** [review]

**Review by SkafteNicki — APPROVED** [review]

**Review by Borda — APPROVED** [review]

**Review by justusschock — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
